src/docs/examples/breadcrumb.html

```diff
@@ -570,9 +570,14 @@ <h2>
               </li>
 
 
-              <li class="qg-share-list-item">
+              <li class="qg-share-list-item twitter-list-item">
                 <a class="qg-share-link" href="https://twitter.com/share?url=https://www.qld.gov.au/health/conditions/health-alerts/coronavirus-lorem ipsum/protect-yourself-others/lorem ipsum" title="Twitter" data-analytics-link-group="qg-sharing-twitter" aria-label="Share on Twitter">
-                  <span class="fa fa-twitter-square fa-2x qg-share-icon" aria-hidden="true"></span>
+                  <span class="x-twitter-square qg-share-icon" aria-hidden="true">
```

Comment:
why is the svg not loaded in css for x-twitter?

Reply:
We tried this using 'content' css rule. However the result was not accurate. The problem was lining up the new svg icon with the the other FA icons.